### PR TITLE
Updated Danish translations

### PR DESCRIPTION
**Pull request by tmlmt**
This PR mostly adds all missing Danish translations, and corrects a couple of minor typos.

Would appreciate a review by the initiator of the Danish translations, @Christian-Schou :)
I am wondering for instance if we should also consider using:
- _ordning_ instead of _abonnement_ in some cases (e.g. _skift ordning_)
- _opsig_/_opsigelse_ instead of _annuller_/_annullering_

### CODE REVIEW

File: ghost/i18n/locales/da/ghost.json [hunk 1]
```diff
@@ -1,27 +1,27 @@
 {
     "All the best!": "Alt det bedste!",
-    "Complete signup for {{siteTitle}}!": "",
+    "Complete signup for {{siteTitle}}!": "Gennemfør tilmelding på {{siteTitle}}!",
```
Comment (Christian-Schou) [inline ghost/i18n/locales/da/ghost.json:3]: What about: **Afslut din oprettelse på {{siteTitle}}!**

File: ghost/i18n/locales/da/portal.json [hunk 2]
```diff
@@ -1,148 +1,148 @@
...
+    "Confirm subscription": "Bekræft abonnement",
+    "Contact support": "Kontakt support",
     "Continue": "Fortsæt",
-    "Continue subscription": "",
-    "Could not sign in. Login link expired.": "",
-    "Could not update email! Invalid link.": "",
-    "Create a new contact": "",
-    "Current plan": "",
+    "Continue subscription": "Fortsæt tilmelding",
+    "Could not sign in. Login link expired.": "Kunne ikke log ind. Log ind link udløbet.",
```
Comment (Christian-Schou) [inline ghost/i18n/locales/da/portal.json:41]: I would change this on to: **Kunne ikke logge find. Dit link til at logge ind med er udløbet.**

File: ghost/i18n/locales/da/portal.json [hunk 3]
```diff
@@ -1,148 +1,148 @@
...
+    "Could not update email! Invalid link.": "Kunne ikke opdatere e-mail! Ugyldigt link.",
+    "Create a new contact": "Opret ny kontakt",
+    "Current plan": "Nuværende abonnement",
     "Delete account": "Slet konto",
-    "Didn't mean to do this? Manage your preferences <button>here</button>.": "",
+    "Didn't mean to do this? Manage your preferences <button>here</button>.": "Mente du ikke at gøre dette? Administrer dine præferencer <button>her</button>.",
```
Comment (Christian-Schou) [inline ghost/i18n/locales/da/portal.json:46]: I would change this one to: **Var det ikke dette du ønskede at gøre? Du kan administrere dine indstillinger <button>her</button>.**

File: ghost/i18n/locales/da/portal.json [hunk 4]
```diff
@@ -1,148 +1,148 @@
...
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "I tilfælde af, at der modtages en permanent fejl ved forsøg på at sende et nyhedsbrev, vil e-mails blive deaktiveret på kontoen.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Tilføj {{senderEmail}} til din kontaktliste i din e-mailklient. Dette signalerer til din e-mailudbyder, at e-mails sendt fra denne adresse skal være tillid til.",
     "Less like this": "Mindre af dette",
-    "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "",
+    "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Sørg for, at e-mails ikke ender i Spam eller Kampagner mapper i din indbakke. Hvis de er, skal du klikke på \"Markér som ikke spam\" og/eller \"Flyt til indbakke\".",
```
Comment (Christian-Schou) [inline ghost/i18n/locales/da/portal.json:75]: I would change "Hvis de er" to "Hvis de gør"

File: ghost/i18n/locales/da/portal.json [hunk 5]
```diff
@@ -1,148 +1,148 @@
...
-    "Subscribe": "",
-    "Subscribed": "",
-    "Success": "",
-    "Success! Check your email for magic link to sign-in.": "",
-    "Success! Your account is fully activated, you now have access to all content.": "",
-    "Success! Your email is updated.": "",
+    "Subscribe": "Tilmeld",
+    "Subscribed": "Tilmeldt",
+    "Success": "Succes",
+    "Success! Check your email for magic link to sign-in.": "Sådan! Tjek din indbakke for et magic link til at logge ind.",
```
Comment (Christian-Schou) [inline ghost/i18n/locales/da/portal.json:112]: magic = magisk? :)

File: ghost/i18n/locales/da/portal.json [hunk 6]
```diff
@@ -1,148 +1,148 @@
...
+    "Success! Your account is fully activated, you now have access to all content.": "Sådan! Din konto er fuldt aktiveret, du har nu adgang til alt indhold.",
+    "Success! Your email is updated.": "Sådan! Din e-mail er opdateret.",
     "Successfully unsubscribed": "Du er nu afmeldt",
     "Thanks for the feedback!": "Tak for din feedback!",
     "That didn't go to plan": "Det gik ikke helt efter planen",
-    "The email address we have for you is {{memberEmail}} — if that's not correct, you can update it in your <button>account settings area</button>.": "",
-    "There was a problem submitting your feedback. Please try again a little later.": "",
+    "The email address we have for you is {{memberEmail}} — if that's not correct, you can update it in your <button>account settings area</button>.": "Den e-mailadresse, vi har til dig, er {{memberEmail}} – hvis det ikke er korrekt, kan du opdatere den i dit <button>kontoindstillingsområde</button>.",
```
Comment (Christian-Schou) [inline ghost/i18n/locales/da/portal.json:118]: Instead of "har til dig" I would say "har på dig"

**Review by Christian-Schou — COMMENTED** [review]
Hey, I have made a few suggestions for some changes in your commits.

**Comment by Christian-Schou** [comment]
As for you question about these words:

- ordning instead of abonnement in some cases (e.g. skift ordning)
- opsig/opsigelse instead of annuller/annullering

**Ordning** is for me a word for something like a service you would get for ex. a company. It could be "frokostordning". I would always go with abonnement as it makes the most sense when discussing subscriptions (for me at least).

opsig / annuller - I think it's the same. If the team behind thinks it better with opsig instead of annuller, that would be no change in my opinion. 

Hope that makes sense 😃

File: ghost/i18n/locales/da/ghost.json (hunk 1 above)
Comment (tmlmt) [reply ghost/i18n/locales/da/ghost.json:3]: I have rarely seen that sentence being used. _Oprettelse af din konto_, sure, but not that one. Have more encountered _tilmelding_

File: ghost/i18n/locales/da/portal.json (hunk 2 above)
Comment (tmlmt) [reply ghost/i18n/locales/da/portal.json:41]: The English original suggests a short form, hence my proposal. I would agree with you if the original contained full sentences

**Comment by tmlmt** [comment]
Thanks for the review :)

I'll take the time later to integrate your suggestions. Ive just left a couple of replies for stuff I don't fully agree with (yet?)

As for the two words:
- agreed about _ordning_. Have seen it used in some online services like Revolut. But Netflix and others use _abonnement_
- in that case I would see that _opsige_ is more used with subscriptions/contracts, like a rental agreement or a monthly subscription (e.g Apple Store). You would more _annullere_ in declaring something not valid (a goal) or withdraw from it (14 day withdrawal right). If you're not convinced, I'm ok leaving it with annuller. I doubt the team has an opinion about Danish :)

**Comment by tmlmt** [comment]
I've now integrated your comments @Christian-Schou in b2ab90d. It'd be great if you could resolve your comments / finalize your review to make this PR ready. Thanks :)

**Comment by Christian-Schou** [comment]
Hi

I will finalize them later today once I get to my laptop 💻

**Review by Christian-Schou — APPROVED** [review]
Approved

**Review by Christian-Schou — APPROVED** [review]
Approved

**Review by Christian-Schou — APPROVED** [review]
Approved

**Comment by daniellockyer** [comment]
Thanks @tmlmt + @Christian-Schou 🕺🏻

**Comment by bobvaneck** [comment]
@tmlmt & @Christian-Schou it looks like line 9 doesn't have a correct translation. It's currently set to:

"Confirm your subscription to {{siteTitle}}": "Annuller dit abonnement hos {{siteTitle}}"

Instead of "confirm" it seems to be saying "cancel". Can one of you review this and suggest a change?

**Comment by Christian-Schou** [comment]
Hi @bobvaneck 👋🏻 

You are correct! I will fix that right away 😃

**Comment by Christian-Schou** [comment]
Hi  @bobvaneck 👋
I have made a new pull request to update the Danish translations: https://github.com/TryGhost/Ghost/pull/19597 😊

**Comment by bobvaneck** [comment]
@Christian-Schou Perfect, thanks for your help Christian. I can see it's already been merged, so it should be in Ghost/Ghost(Pro) soon.

**Pull request merged**